packages/git/src/browser/diff/git-diff-widget.tsx

```diff
@@ -32,6 +32,8 @@ export const GIT_DIFF = 'git-diff';
 @injectable()
 export class GitDiffWidget extends GitNavigableListWidget<GitFileChangeNode> implements StatefulWidget {
 
+    protected readonly GIT_DIFF_TITLE = 'Git Diff';
```

Comment:
I'm not sure about naming. Was there a reason that we had decided to stick with `Diff` instead of `Git Diff`, for instance it should be used no in Git context? @kittaakos @jbicker @svenefftinge

Reply:
@vince-fugnitto can we keep it `Diff`, unfortunately noone can remember, i guess it was implemented in context of compare feature and supposed to be used for other SCMs

Reply:
@akosyakov sure I can keep it as `Diff` :)